src/components/organisms/StoryList.js

```diff
@@ -1,26 +1,28 @@
 import React from 'react';
 import PropType from 'prop-types';
+import LinkCard from './LinkCard';
+import { useAuthContext } from '../../contexts/AuthContext';
+
 import './StoryList.scss';
 
-const StoryList = ({ stories }) => (
-  <section className="story-list">
-    {
-      stories.map((story) => (
-        <article key={story.id}>
-          <header>
-            <h3>{story.title}</h3>
-            <span className="byline">
-              {`by ${story.author}`}
-            </span>
-          </header>
-          <p>
-            {story.content}
-          </p>
-        </article>
-      ))
-    }
-  </section>
-);
+const StoryList = ({ stories }) => {
+  const { currentUser } = useAuthContext();
+
+  return (
+    <div className="story-list-component">
+      <section className="story-list">
+        {
+          stories.map((story) => (
+            <LinkCard story={story} key={story.id} />
+          ))
+        }
+      </section>
+      <article className="user-info-panel">
```

Comment:
semantically, this wouldn't be an article. I would change this to a section tag instead.

Reply:
oops, i don't think i pushed my most recent changes. this was supposed to be changed. i created a UserPanel component that is supposed to be here